### ISSUE TITLE
Data screen with no pictures

### ISSUE THREAD

**Issue by tsegner** [FirewallDragon/amulet-of-entropy#3]
It took some work, but I have this up and running... kind of.
​It boots to a screen showing random scrolling data for items in a list (LDR:, Temp:, Shot, GyrX, etc) but just stays there no matter what I do.
​Any advice is Appreciated.
![dd](https://user-images.githubusercontent.com/34924362/200675828-ddabc25b-f710-4083-be11-3df9eda7f80f.jpg)

**Comment by FirewallDragon** [comment]
This is the "hb80_demo.ino" test program from step 10 on Instructibles. You need to load the amulet.ino from this repo, which is explained in the Setup doc.

**Comment by tsegner** [comment]
Thank you for this super-fast response!
So, it looks like the amulet.ino is failing to complete the upload and the pico is reverting back to its last image?

A recompile fix it!
Awesome project, please keep up the great work sir!

**Comment by FirewallDragon** [comment]
Yes, that appears to be the issue. However, I'm not sure what to do about that one. Sounds like a more general RPi or Arduino IDE issue. You can try standard stuff like restarting the IDE, resetting the RPi, or restarting your PC.

**Comment by tsegner** [comment]
A recompile fix it!
Awesome project, please keep up the great work sir!

**Comment by FirewallDragon** [comment]
Thanks!! Enjoy!
